### PR TITLE
rsz: use masters for report_dont_use to avoid listing cells by corner

### PR DESCRIPTION
**Pull request by gadfort**
Changes:
- use masters to uniquely report dont use cells, otherwise one per corner appears in the report

### CODE REVIEW

File: src/rsz/src/Resizer.cc [hunk 1]
```diff
@@ -1810,8 +1810,14 @@ void Resizer::reportDontUse() const
   if (dont_use_.empty()) {
     logger_->report("  none");
   } else {
+    std::set<dbMaster*> cells;
+
     for (auto* cell : dont_use_) {
-      logger_->report("  {}", cell->name());
+      cells.insert(db_network_->staToDb(cell));
```
Comment (povik) [inline src/rsz/src/Resizer.cc:1816]: I think `if(isLinkCell(cell))` also filters out the per-corner copies

Comment (maliberty) [reply src/rsz/src/Resizer.cc:1816]: @gadfort this seems correct

Comment (gadfort) [reply src/rsz/src/Resizer.cc]: done

File: src/rsz/src/Resizer.cc [hunk 2]
```diff
@@ -1803,14 +1803,17 @@ bool Resizer::dontUse(LibertyCell* cell)
   return cell->dontUse() || dont_use_.hasKey(cell);
 }
 
-void Resizer::reportDontUse() const
+void Resizer::reportDontUse()
```
Comment (maliberty) [inline src/rsz/src/Resizer.cc:1806]: I think you can make Resizer::isLinkCell const rather than drop const here.

**Review by maliberty — APPROVED** [review]

**Pull request merged**